src/serializers/decoders/current/v15/decode_graph_entities.c

```
@@ -224,6 +224,6 @@ void RdbLoadDeletedEdges_v15
 	// edge id X N
 	for(uint64_t i = 0; i < deleted_edge_count; i++) {
 		EdgeID id = SerializerIO_ReadUnsigned(rdb);
-		Serializer_Graph_MarkEdgeDeleted(gc->g, id);
+		Graph_MarkEdgeDeleted(gc->g, id);
```

cppcheck

[error] Null pointer dereference

(nullPointer)